itenium-socks/src/app/socks/sock.component.ts

```diff
@@ -3,17 +3,27 @@ import { Observable } from 'rxjs';
 import { Sock } from './sock.model';
 import { SocksService } from './socks.service';
 import { AsyncPipe, NgIf, TitleCasePipe } from '@angular/common';
+import {FormControl, FormGroup, ReactiveFormsModule, Validators} from "@angular/forms";
 
 @Component({
   selector: 'app-sock',
   standalone: true,
-  imports: [NgIf, AsyncPipe, TitleCasePipe],
-  templateUrl: './sock.component.html'
+  imports: [NgIf, AsyncPipe, TitleCasePipe, ReactiveFormsModule],
+  templateUrl: './sock.component.html',
 })
 export class SockComponent {
   sock$!: Observable<Sock>;
+  reviewForm: FormGroup;
+  formError: boolean = false;
+  apiError: boolean = false;
+  apiSuccess: boolean = false;
 
-  constructor(private socksService: SocksService) {}
+  constructor(private socksService: SocksService) {
+    this.reviewForm = new FormGroup({
+      'review': new FormControl(null, Validators.required),
+      'rating': new FormControl('1')
+    });
```

Comment:
Tegenwoordig kan je dit ook allemaal strongly typed maken (wat vb handig is bij `.valueChanges` etc

```
rating: new FormControl<number>(1)
```